### PR TITLE
refactor: redesign create-room page

### LINKED ISSUE #71: Admin create room page
### Tasks

# Admin create room page

## Why do we need this?

We want to transition to a more innovative and user-friendly design compared to the old one, aiming to increase user attention.

## Implementation Considerations

- Pay attention to the use of CSS variables in the codebase.
- Ensure proper naming conventions for properties such as color, font-size, or font-weight when adding them to the variable.scss file.

## Scope

- Current design should be replaced by the new one.

## Design Link

[View Design](https://www.figma.com/file/3Rvo2R5AWAse3O09KtuhZT/GurubuV2?type=design&node-id=71-4667&mode=design&t=ZA9MrkJBhz7y2M3b-0)

## Questions or Issues

If you still have questions, feel free to comment within the relevant issue. You can tag @armagandalkiran and @aahmetcakir for assistance.

### PR DESCRIPTION
**Pull request by ethsmaa**
Fixes #71

### CODE REVIEW

**Comment by armagandalkiran** [comment]
Thank you! I just left a comment other than that LGTM! @aahmetcakir will do UAT ( User Acceptance Testing ) and after that we can merge it.

**Comment by mertcanaltin** [comment]
LGTM!

**Review by mertcanaltin — APPROVED** [review]

**Comment by mertcanaltin** [comment]
a few improvements are needed on the uat side 🎉

**Comment by armagandalkiran** [comment]
Can you see the comments in vercel? @ethsmaa

**Comment by ethsmaa** [comment]
> Can you see the comments in vercel? @ethsmaa

yes, i can see it now. I will deal with it as soon as possible

**Comment by aahmetcakir** [comment]
Hi @ethsmaa, I commented little few things. You can see on vercel preview feature branch. If could't see reach me. Thank you for contribute 🚀

**Comment by ethsmaa** [comment]
The 'joinMode' prop is no longer used. I found that keeping 'groomingType' as '0' and '1' was not descriptive enough. I have adjusted the frontend accordingly, and the backend can be updated accordingly as well.

File: gurubu-client/src/app/components/room/nickname-form.tsx
Comment (hsglc) [inline gurubu-client/src/app/components/room/nickname-form.tsx:125]: Hey @ethsmaa, I think usage of useMemo here couldn't be necessary. The calculation of this function is not expensive. Maybe we can create a normal function and call it in the return section.
[useMemo](https://react.dev/reference/react/useMemo)

**Comment by armagandalkiran** [comment]
I just left 2 suggestions, other than that LGTM! Thank you so much for your contribution 🚀

File: gurubu-client/src/app/components/room/nickname-form.tsx
Comment (ethsmaa) [reply gurubu-client/src/app/components/room/nickname-form.tsx:125]: thanks! i will fix it ✨

**Comment by ethsmaa** [comment]
> I just left 2 suggestions, other than that LGTM! Thank you so much for your contribution 🚀

hey, I can't see a comment different from the ones posted on vercel about 10 hours ago

File: gurubu-client/src/app/components/room/nickname-form.tsx
Comment (armagandalkiran) [reply gurubu-client/src/app/components/room/nickname-form.tsx:20]: @ethsmaa

File: gurubu-client/src/app/components/room/nickname-form.tsx
Comment (armagandalkiran) [reply gurubu-client/src/app/components/room/nickname-form.tsx:169]: @ethsmaa

File: gurubu-client/src/app/components/room/nickname-form.tsx
Comment (armagandalkiran) [reply gurubu-client/src/app/components/room/nickname-form.tsx:53]: @ethsmaa

**Comment by armagandalkiran** [comment]
@ethsmaa I tagged you can you see them now ?

File: gurubu-client/src/app/components/room/nickname-form.tsx
Comment (armagandalkiran) [reply gurubu-client/src/app/components/room/nickname-form.tsx:54]: thank you!

**Comment by armagandalkiran** [comment]
this pr is LGTM to me! @aahmetcakir If UAT is ok too this pr is ready to merge 🚀

**Review by bufgix — APPROVED** [review]
Dont mind me. Im just approved my sis 🎉

**Review by mertcanaltin — DISMISSED** [review]
lgtm!

File: gurubu-client/src/app/styles/room/nickname-form.scss
Comment (aahmetcakir) [inline gurubu-client/src/app/styles/room/nickname-form.scss:134]: LGTM! I resolved other comments just ı saw this can we add font-family inter into `__button` after that we can merge

**Review by armagandalkiran — APPROVED** [review]
LGTM! thank you for your contribution! 🚀

**Pull request merged**
